### ISSUE TITLE
Clicking on search engine icon minimizes dismisses keyboard and Android nav buttons

### ISSUE THREAD

**Issue by eliserichards** [mozilla-mobile/fenix#15357]
## Steps to reproduce
1. Click on toolbar to open search dialog
2. Click on the search engine icon on the far left of the toolbar

<img width="380" alt="image" src="https://user-images.githubusercontent.com/43795363/94061154-9e47e180-fdaa-11ea-8d25-ed9b310fcbd6.png">

### Expected behavior
Nothing happens (or it opens search engines) - **what do we want the behavior to be??**

### Actual behavior
Search dialog is minimized and bottom back/home/app switch icons disappear

Video: 
[20200923_122433.mp4.zip](https://github.com/mozilla-mobile/fenix/files/5271378/20200923_122433.mp4.zip)

### Device information

* Android device: Pixel 3 android 10
* Fenix version: nightly 81.1.2 (master)

**Comment by MReschenberg** [comment]
hello! from an a11y perspective, I worry about touch target size. We want to make sure users (especially those with motor disabilities) are able to easily touch & activate both the button and the input for the search bar separately. Looks like the minimum target size for android is something like 48x48, will the search button/dropdown meet that minimum?

**Comment by violasong** [comment]
I realized the [WCAG requirement](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) is actually 44x44, which is great for this design, since the buttons are much less important than the central input area. So I'm planning to propose this, with potential extra tap area on the edges.

<img width="370" alt="image" src="https://user-images.githubusercontent.com/55994/95927764-8a254d80-0d74-11eb-8d4d-845ce139bfd6.png">

Some very helpful comments from Twitter :)
- [Florens:](https://twitter.com/fvsch/status/1316091598514651137) I like it. It's actually clearer to me like this. In the current UI, the "Search engine" button is disconnected from the input, so it's not 100% clear to me where it goes or what it does.
- [Julie:](https://twitter.com/juliemoynat/status/1316095948494438400) I love and need this feature! When the address bar is at the top of the screen, the "search engine" button is too far away and this is not intuitive at all. I always forget it…
- [Jeppe:](https://twitter.com/Tenzer/status/1316099184475475972) I'll admit I've previously have been pressing the logo in an attempt to switch the search engine, so this would be a welcome change in my mind

**Comment by MReschenberg** [comment]
Ah, yeah the guidance I was referring to is specific for android, which differs from the WCAG advice -- [you can find it here](https://support.google.com/accessibility/android/answer/7101858?hl=en).

**Comment by violasong** [comment]
> Ah, yeah the guidance I was referring to is specific for android, which differs from the WCAG advice -- you can find it here.

Thanks Morgan - yes, I'm generally striving for this Android standard, but we think the WCAG minimum works better in this particular instance due to the concern that the buttons start to visually crowd the much more important input area. (People accidentally tapping the QR code when they mean to tap the input is a bigger problem than vice-versa.)

**Comment by hmble** [comment]
It would be nice if we can also assign shortuct key string for search engines like we have in firefox desktop
https://support.mozilla.org/en-US/kb/assign-shortcuts-search-engines

I use "d" as a shortuct key for duckduckgo which automatically change seadch engine to duckduckgo.

**Comment by violasong** [comment]
Same basic design concept created by Bram 2 months ago :) https://github.com/mozilla-mobile/fenix/issues/12619

Current [Figma file](https://www.figma.com/file/Sxi1ztO52VJ4mRaCQilrAw/Fenix-address-bar-simplification?node-id=1%3A2) for the new design

**Comment by violasong** [comment]
[Figma file](https://www.figma.com/file/Sxi1ztO52VJ4mRaCQilrAw/Fenix-address-bar-simplification?node-id=1%3A2) now reflects the final version for this hand-off.

Future sprints could include:
- Collect telemetry to determine if some things should be in a ••• menu, or more prominent
- Improve search engine panel
- Shortcut key strings

**Comment by ekager** [comment]
Part of the work was completed already in AC by Jeff: https://github.com/mozilla-mobile/android-components/pull/8640

**Comment by eliserichards** [comment]
Followups: #16659 #16658
